src/mock.rs

```diff
@@ -0,0 +1,87 @@
+use crate as peaq_storage;
+use frame_support::{
+	parameter_types,	
+};
+use frame_system as system;
+use pallet_timestamp;
+use sp_core::{sr25519, H256};
+use sp_runtime::{
+    testing::Header,
+    traits::{BlakeTwo256, IdentityLookup},
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+    pub enum Test where
+        Block = Block,
+        NodeBlock = Block,
+        UncheckedExtrinsic = UncheckedExtrinsic,
+    {
+        System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
```

Comment:
Does the storage pallet have a dependency on the Timestamp?

Reply:
I do not think so. Let me work it out.

Reply:
Then please remove all the Timestamp-related stuff in the mock file and Cargo.toml.

Reply:
Done